### PR TITLE
fix: try get image url from not only scr attribute

### PR DESCRIPTION
**Pull request by sergienko88**

### CODE REVIEW

File: UnifyEngine/FulldiveVRVideoProvidersUnifyEngineTests/Implementation/DocumentInterface.cs [hunk 1]
```diff
@@ -33,7 +34,22 @@ public IEnumerable<string> GetImagesByCssQuery(string query)
         {
             return this._htmlDocument.DocumentNode.CssSelect(query)
                 .Where(x => x != null && x.HasAttributes)
-                .Select(x => x.GetAttributeValue("src"));
+                .Select(x => GetUrlImageArributeValue(x, _possibleImageUrlAttributeName);
+        }
+
+        private string GetUrlImageArributeValue(HtmlNode htmlNode, string[] altAttributeNames)
```
Comment (alex-y-su) [inline UnifyEngine/FulldiveVRVideoProvidersUnifyEngineTests/Implementation/DocumentInterface.cs]: Can be static

File: UnifyEngine/FulldiveVRVideoProvidersUnifyEngineTests/Implementation/DocumentInterface.cs [hunk 2]
```diff
@@ -33,7 +34,22 @@ public IEnumerable<string> GetImagesByCssQuery(string query)
...
+        {
+            string attributeValue = string.Empty;
+            for (int i = 0; i < altAttributeNames.Length; i++)
```
Comment (alex-y-su) [inline UnifyEngine/FulldiveVRVideoProvidersUnifyEngineTests/Implementation/DocumentInterface.cs]: replace by foreach

File: UnifyEngine/FulldiveVRVideoProvidersUnifyEngineTests/Implementation/DocumentInterface.cs (hunk 1 above)
Comment (sergienko88) [reply UnifyEngine/FulldiveVRVideoProvidersUnifyEngineTests/Implementation/DocumentInterface.cs]: 37 :  .Select(x => GetUrlImageArributeValue(x, _possibleImageUrlAttributeName)); //mistake. Added ")" 
40 : private static string GetUrlImageArributeValue(HtmlNode htmlNode, string[] altAttributeNames)

File: UnifyEngine/FulldiveVRVideoProvidersUnifyEngineTests/Implementation/DocumentInterface.cs (hunk 2 above)
Comment (sergienko88) [reply UnifyEngine/FulldiveVRVideoProvidersUnifyEngineTests/Implementation/DocumentInterface.cs]: Replaced to foreach loop

**Pull request merged**